src/main/java/de/danielbechler/diff/node/DiffNode.java

```diff
@@ -468,6 +468,20 @@ public final void visitParents(final Visitor visitor)
 		return null;
 	}
 
+   public <T extends Annotation> T getAnnotation(final Class<T> annotationClass) {
+      if (accessor instanceof PropertyAwareAccessor) {
+         return ((PropertyAwareAccessor) accessor).getAnnotation(annotationClass);
+      }
+      return null;
+   }
+
+   public int getFieldModifiers() {
```

Comment:
As explained in my other comment, this method should not become a part of the public API.